src/neo4j_graphrag/retrievers/hybrid.py

```diff
@@ -193,8 +194,11 @@ def get_search_results(
             embedding_node_property=self._embedding_node_property,
             neo4j_version_is_5_23_or_above=self.neo4j_version_is_5_23_or_above,
         )
-
-        logger.debug("HybridRetriever Cypher parameters: %s", parameters)
+        sanitized_parameters = copy.deepcopy(parameters)
+        sanitized_parameters["query_vector"] = (
```

Comment:
Looks good. I'm just wondering if that would be misleading to display the `query_vector` as parameter (even as None) when it's not in the actual parameters? Maybe I'm just overthinking :D

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/192/commits/64dc99d15149f998a94807eed677d1bd043a9ca1

I think you're right, it's misleading